src/transforms/filter.jl

```diff
@@ -69,22 +65,45 @@ DropMissing(cols::T...) where {T<:ColSelector} =
 
 isrevertible(::Type{<:DropMissing}) = true
 
-_ftrans(::DropMissing{Colon}, table) =
+# ftrans
+_ftrans(::DropMissing{Colon}, cols) =
   Filter(row -> all(!ismissing, row))
 
-function _ftrans(transform::DropMissing, table)
-  allcols = Tables.columnnames(table)
-  cols = _filter(transform.colspec, allcols)
+_ftrans(::DropMissing, cols) =
   Filter(row -> all(!ismissing, getindex.(Ref(row), cols)))
-end
+
+# nonmissing 
+_nonmissing(::Type{T}, c) where {T} = c
+_nonmissing(::Type{Union{Missing,T}}, c) where {T} =
+  collect(T, c)
+
+_nonmissing(col) = _nonmissing(eltype(col), col)
 
 function apply(transform::DropMissing, table)
-  ftrans = _ftrans(transform, table)
+  colnames = Tables.columnnames(table)
+  select = _filter(transform.colspec, colnames)
```

Comment:
Maybe rename this to selnames to  indicate that these are still names of columns.

Reply:
OK

Reply:
Done with [7db5e28](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/7db5e28ea9f808a8a4a1f1426f7ccf7b866f034c)